lein test luhn-test

Ran 1 tests containing 13 assertions.
0 failures, 0 errors.



>lein repl

REPL-y 0.3.7, nREPL 0.2.12
Clojure 1.10.0
Java HotSpot(TM) 64-Bit Server VM 14.0.1+7
    Docs: (doc function-name-here)
          (find-doc "part-of-name-here")
  Source: (source function-name-here)
 Javadoc: (javadoc java-object-or-class-here)
    Exit: Control+D or (exit) or (quit)
 Results: Stored in vars *1, *2, *3, an exception in *e

=> (require 'luhn-test)
WARNING: replace already refers to: #'clojure.core/replace in namespace: luhn, being replaced by: #'clojure.string/replace
nil

=> (clojure.test/run-tests 'luhn-test)

Testing luhn-test

Ran 1 tests containing 13 assertions.
0 failures, 0 errors.
{:test 1, :pass 13, :fail 0, :error 0, :type :summary}

=> Bye for now!



>lein repl

REPL-y 0.3.7, nREPL 0.2.12
Clojure 1.10.0
Java HotSpot(TM) 64-Bit Server VM 14.0.1+7

user=> (require 'luhn-test)
WARNING: replace already refers to: #'clojure.core/replace in namespace: luhn, being replaced by: #'clojure.string/replace
nil

user=> (clojure.test/run-tests 'luhn-test)

Testing luhn-test

Ran 13 tests containing 13 assertions.
0 failures, 0 errors.
{:test 13, :pass 13, :fail 0, :error 0, :type :summary}

user=> Bye for now!
